navigation.js

```diff
@@ -0,0 +1,52 @@
+const links = document.querySelectorAll('.nav-link');
+
+links.forEach((link) => {
+  link.addEventListener('click', (e) => {
+    e.preventDefault();
+    const id = link.getAttribute('href').slice('1');
+
+    const currentSection = document.getElementById(id);
+    const blockElement = document.querySelector('.block');
+    blockElement.classList.remove('block');
+    blockElement.classList.add('none');
+    currentSection.classList.remove('none');
+    currentSection.classList.add('block');
+  });
+});
+
+function checkTime(i) {
+  if (i < 10) {
+    i = `0${i}`;
+  }
+  return i;
+}
+
+function startTime() {
+  const monthNames = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+  const today = new Date();
+  const k = today.getMonth();
+  const l = today.getDate();
+  const n = today.getFullYear();
+  const h = today.getHours();
+  let m = today.getMinutes();
+  let s = today.getSeconds();
+  m = checkTime(m);
+  s = checkTime(s);
+  document.getElementById('time').innerHTML = `${monthNames[k]} ${l} ${n}, ${h}:${m}:${s}`;
+  setTimeout(startTime, 1000);
```

Comment:
- [ ] For setting the time you were supposed to use the luxon library. However, I have noticed that the link is not working on the project requirements but we apologize for the inconvenience. Please refer to this [article](https://moment.github.io/luxon/#/) on how to set it up as the broken link gets updated. Unfortunately, this is a hard requirement for the project and needs to be implemented.

Reply:
Thank you for your reviews. I added luxon library to the page.